src/test/plugins/mvcc_delete_plugin_system_test.cpp

```diff
@@ -158,7 +158,7 @@ TEST_F(MvccDeletePluginSystemTest, CheckPlugin) {
 
   // (5) Wait for the MvccDeletePlugin to delete chunk 2 logically
   {
-    auto attempts_remaining = 10;
+    auto attempts_remaining = 20;
```

Comment:
Put a constant somewhere?

Reply:
Maybe simply skip this test with asan?

Reply:
For now, raising `attempts_remaining` to 100 should help. If we ever run into this issue again, skipping the test would be our best option.